apps/src/sites/studio/pages/projects/index.js

```diff
@@ -28,10 +29,11 @@ $(document).ready(() => {
     url: `/api/v1/projects/gallery/public/all/${MAX_PROJECTS_PER_CATEGORY}`,
     dataType: 'json'
   }).done(projectLists => {
+    getStore().dispatch(setProjectLists(projectLists));
```

Comment:
nit: This method calls `getStore` a bunch of times. I might just have a `const store = getStore();` at the top of the method. Any perf impact should be negligible, but feels slightly cleaner.